ausbills/parliament/federal.py

```diff
@@ -376,15 +362,66 @@ def data(self):
                     self._bill_data[TEXT_LINK + '_' +
                                     TEXT] = self.bill_text_links[reading][TEXT]
                     self._bill_data[CURRENT_READING] = reading
-            self._bill_data[EM_LINK + '_' +
-                            TEXT] = self.explanatory_memoranda_links[TEXT]
+            try:
+                self._bill_data[EM_LINK + '_' +
```

Comment:
not sure where `self._bill_data` is defined. This may be unsafe

Reply:
`self._bill_data`'s values are defined by that function, since it's declared as a `dict` at the start of `BillFedHelper`